src/plugins/intel_cpu/src/nodes/executors/transpose.cpp

```diff
@@ -0,0 +1,7 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "transpose.hpp"
+
+ov::intel_cpu::TransposeExecutor::TransposeExecutor(const ov::intel_cpu::ExecutorContext::CPtr context) : context(context) {}
```

Comment:
Just curious, why can't we declare this in the header?

Reply:
I assume, we need to follow file structure implemented by Dmitry in ARM PoC PR: 
eltwise: https://github.com/openvinotoolkit/openvino/pull/15256/files#diff-827ef1270fb5468920a250dad546cf597e6826e607b7c77c4e7f1677e9eb0dde
pooling: https://github.com/openvinotoolkit/openvino/pull/15256/files#diff-0095ace808b9e325324f384711afab879186aadde53fe5c3ccd55a18a2e99b4b